lib/tasks/webpacker/verify.rake

```diff
@@ -1,17 +0,0 @@
-require "webpacker/configuration"
-
-namespace :webpacker do
-  namespace :install do
-    desc "Verifies if webpacker is installed"
-    task :verify do
-      begin
-        File.read(Webpacker::Configuration.file_path)
```

Comment:
But hey, I'm not firm on this. I get your point too. Maybe wait for others to comment?